test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -53,7 +53,7 @@ def run_smdebug_test(
     ec2_connection.run(f"$(aws ecr get-login --no-include-email --region {region})", hide=True)
 
     ec2_connection.run(
-        f"{docker_executable} run --name {container_name} -v "
+        f"{docker_executable} run --shm-size=1g --name {container_name} -v "
```

Comment:
Please make the addition of `--shm-size=1g` optional, based on whether the instance type is p2.8xlarge or any other instance type where it is known to be necessary.